lightning/src/chain/mod.rs

```diff
@@ -23,6 +23,8 @@ pub mod chainmonitor;
 pub mod channelmonitor;
 pub mod transaction;
 pub mod keysinterface;
+pub(crate) mod onchaintx;
+pub(crate) mod onchain_utils;
```

Comment:
"chain" seems redundant in these names after the move, though maybe not worth changing right now.

Reply:
Well switched to "package.rs" so it's a bit better.